DEPRECATIONS.md

```diff
@@ -120,13 +118,11 @@ All filters
 - sepia
 - backdrop-filter
 - backdrop-filter-none
-- backdrop-blur-sm
-- backdrop-blur
-- backdrop-blur-md
-- backdrop-blur-lg
-- backdrop-blur-xl
-- backdrop-blur-2xl
-- backdrop-blur-3xl
+- backdrop-blur-sm :: prefer backdrop-blur-s
+- backdrop-blur-md :: prefer backdrop-blur-m
+- backdrop-blur-lg :: prefer backdrop-blur-l
+- backdrop-blur-2xl :: prefer backdrop-blur-xxl
+- backdrop-blur-3xl :: prefer backdrop-blur-xxxl
```

Comment:
Nice 👍 Will we need to update the messages in preset-migrate with these new recommendations

Reply:
I'll fix.